Logger/FileLogger.cs

```diff
@@ -0,0 +1,23 @@
+#nullable enable
+using System;
+using System.IO;
+
+namespace Logger;
+
+public class FileLogger : BaseLogger
+{
+    public string PathToFile {get; set; }
+
+    public FileLogger(string pathToFile)
+    {
+        PathToFile = pathToFile;
+    }
+
+    override public void Log(LogLevel logLevel, string message)
+    {
+        string currentDateAndTime = DateTime.Now.ToString("MM/d/yyy HH:mm:ss tt");
```

Comment:
I believe the way we resolved this error was with the following:
```suggestion
            string currentDateTime = DateTime.Now.ToString("MM/dd/yyyy hh:mm:ss tt", CultureInfo.CurrentCulture);
```

Reply:
I think the formatting of the DateTime was wrong. I didn't have to use CultureInfo.CurrentCulture in my build

Reply:
Thanks!

Reply:
@pja6 @MattA41 That's a good point out. `CultureInfo.CurrentCulture` isn't necessary in this instance, but it's something to keep in mind when it comes to localization. Refer to the [following link](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.currentculture?view=net-8.0) for more information.